### DISCUSSION TITLE
Windows 11 22H2版本下有错位现象

### DISCUSSION THREAD

**Discussion by Ytimes2** [zhongyang219/TrafficMonitor#1205] [General]
在最新的22H2版本里试了一下，软件有错位现象，会往任务栏中间去靠近，无法显示在右边

**Comment by keithop** [comment]
same here

**Reply by Ytimes2** [reply]
#1106 This is a temporary fix

**Comment by 0ne2free** [comment]
一样。期待更新

**Reply by liwuxcn** [reply]
期待大神修正一下，最近看不到它感觉电脑里少点东西。
<img width="187" alt="屏幕截图 2022-07-30 102923" src="https://user-images.githubusercontent.com/15944220/181866842-80500889-1093-4d42-9e38-fa01654a4f4d.png">

**Reply by AliRao47** [reply]
![Uploading image.png…]()
same error after updating to 22H2 unable to align right side.

**Reply by AliRao47** [reply]
![Uploading image.png…]()
same error after updating to 22H2 unable to align right side.

**Reply by AliRao47** [reply]
![image](https://user-images.githubusercontent.com/109412365/191823836-82c83679-8b25-4c63-ad7a-58f71cf5cce2.png)
same error after updating to 22H2 unable to align right side.

**Reply by AliRao47** [reply]
![image](https://user-images.githubusercontent.com/109412365/191823836-82c83679-8b25-4c63-ad7a-58f71cf5cce2.png)
same error after updating to 22H2 unable to align right side.

**Reply by AliRao47** [reply]
![Uploading image.png…]()
same error after updating to 22H2 unable to align right side.

**Reply by AliRao47** [reply]
![image](https://user-images.githubusercontent.com/109412365/191823836-82c83679-8b25-4c63-ad7a-58f71cf5cce2.png)
same error after updating to 22H2 unable to align right side.

**Reply by AliRao47** [reply]
![image](https://user-images.githubusercontent.com/109412365/191823836-82c83679-8b25-4c63-ad7a-58f71cf5cce2.png)
same error after updating to 22H2 unable to align right side.

**Reply by AliRao47** [reply]
![image](https://user-images.githubusercontent.com/109412365/191823836-82c83679-8b25-4c63-ad7a-58f71cf5cce2.png)
same error after updating to 22H2 unable to align right side.

**Comment by ZYXCXDX** [comment]
yup,,same please fixx

**Comment by JimLambda** [comment]
Same here! Please fix this! Thanks!
遇到了一样的问题！希望大神能修复！多谢！

**Comment by shaoyouvip** [comment]
是的 我升级后也遇到 希望大佬能修复  谢谢！！！

**Comment by RR25amz** [comment]
同样问题+1 win11 22H2

**Comment by injainja** [comment]
Same here :-)

**Comment by SmallFang-letsgo** [comment]
遇到一样的问题了 期待作者早点修复

**Comment by lovepepper** [comment]
i got the same problem, hope solve soon

**Reply by Niko-01** [reply]
正式版22H2也存在错位问题，期待作者早日修复

**Comment by adammuizweb** [comment]
Is there any update ?

**Comment by LgenedX** [comment]
is it fixed ?

**Comment by Niko-01** [comment]
no



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年9月26日(周一) 凌晨1:11
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [zhongyang219/TrafficMonitor] Windows 11 22H2版本下有错位现象 (Discussion #1205)




 
is it fixed ?

**Comment by smartpippin** [comment]
![QQ截图20220927092039](https://user-images.githubusercontent.com/22493619/192409407-f9e338eb-8b70-49c7-ad42-b867dfbb11b4.jpg)
win11 22h2有同样的问题

**Comment by injainja** [comment]
https://github.com/zhongyang219/TrafficMonitor/pull/1106

Works for me

**Reply by vishalcore** [reply]
hey, are u able to download the fix?

**Comment by lovepepper** [comment]
Dear all，
开发者已经更新了，只不过还没有release安装包，我打包一下供大家下载。
如果有任何设计开发者个人利益问题，请告知我，我删除链接。

百度云：链接：https://pan.baidu.com/s/13RIHPZPhRu6zUM62YbDOXA?pwd=3jdk
提取码：3jdk
阿里云：https://www.aliyundrive.com/s/M9Fj1TBn2cH

阿里云因为云盘限制，上传的文件经过转换，需要使用winRAR或者7zip等解压软件打开，不可使用WIN11资源管理器直接打开。



阿里云：https://www.aliyundrive.com/s/TdCXXwj4L9N
从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: Fahad Bin Hasan ***@***.***>
发送时间: 2022年9月29日 2:03
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [zhongyang219/TrafficMonitor] Windows 11 22H2版本下有错位现象 (Discussion #1205)


How to fix the issue?

―

**Reply by injainja** [reply]
For both links I need to register to download - can anyone provide a download link where we do not need to register?

**Reply by lovepepper** [reply]
Maybe you can download from here, but very slow

TrafficMonitor_V1.83_x64.zip: https://url23.ctfile.com/f/13898623-682152071-a1e356?p=2935 （访问密码：2935）

**Reply by vishalcore** [reply]
i have downloded the file, what should I do next?

**Reply by injainja** [reply]
> Vielleicht können Sie von hier herunterladen, aber sehr langsam
> 
> TrafficMonitor_V1.83_x64.zip: https://url23.ctfile.com/f/13898623-682152071-a1e356?p=2935（访问密码：2935）
Works! Thank you.

**Reply by injainja** [reply]
> Ich habe die Datei heruntergeladen, was soll ich als nächstes tun?

Unzip it and install it as usual.

**Reply by 04SUAF04** [reply]
> > Vielleicht können Sie von hier herunterladen, aber sehr langsam
> > TrafficMonitor_V1.83_x64.zip: [https://url23.ctfile.com/f/13898623-682152071-a1e356?p=2935（访问密码：2935）](https://url23.ctfile.com/f/13898623-682152071-a1e356?p=2935%EF%BC%88%E8%AE%BF%E9%97%AE%E5%AF%86%E7%A0%81%EF%BC%9A2935%EF%BC%89)
> > Works! Thank you.

Replacement .exe gives me all sorts of missing dll errors.  Have reinstalled all runtimes, but no dice

**Comment by cgonIT** [comment]
Same probelm, and as im seen, no fix yet... uninstalling until the creator answer here...now it's unusable, so sad.

**Comment by lovepepper** [comment]
Just unzip and replace the old one on you pc, and you will find it works.

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: ***@***.***>
发送时间: 2022年9月30日 14:07
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [zhongyang219/TrafficMonitor] Windows 11 22H2版本下有错位现象 (Discussion #1205)


Ich habe die Datei heruntergeladen, was soll ich als nächstes tun?

Unzip it and install it as usual.

**Comment by arunpts** [comment]
Any fixes yet?

**Comment by lovepepper** [comment]
Hi,

Maybe you can download from here, but very slow

TrafficMonitor_V1.83_x64.zip: https://url23.ctfile.com/f/13898623-682152071-a1e356?p=2935 （访问密码：2935）

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: ***@***.***>
发送时间: 2022年9月29日 12:59
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [zhongyang219/TrafficMonitor] Windows 11 22H2版本下有错位现象 (Discussion #1205)


For both links I need to register to download - can anyone provide a download link where we do not need to register?

―

**Reply by arunpts** [reply]
the new update is working very well...thank you🙂
